src/window.rs

```diff
@@ -1389,3 +1390,27 @@ impl Default for UserAttentionType {
         UserAttentionType::Informational
     }
 }
+
+/// A window level groups windows with respect to their z-position.
+///
+/// The relative ordering between windows in different window levels is fixed.
+/// The z-order of a window within the same window level may change dynamically on user interaction.
+///
+/// ## Platform-specific
+///
+/// - **iOS / Android / Web / Wayland:** Unsupported.
+#[derive(Debug, PartialEq, Eq, Clone, Copy)]
+pub enum WindowLevel {
+    /// The window will always be below normal windows.
+    AlwaysOnBottom,
```

Comment:
It is not immediately clear why you would want to use this level, could we add use-cases for this to the documentation?

Reply:
It is hard for me to come up with a good use-case but I think this will be useful for a widgets app.